cmd/argocd/commands/app.go

```diff
@@ -2400,7 +2400,7 @@ func waitOnApplicationStatus(ctx context.Context, acdClient argocdclient.Client,
 				_ = w.Flush()
 			}
 		case "tree":
-			mapUidToNode, mapParentToChild, parentNode, mapNodeNameToResourceState := resourceParentChild(ctx, acdClient, appName, appNs)
+			mapUidToNode, mapParentToChild, parentNode, mapNodeNameToResourceState := resourceParentChild(ctx, acdClient, appRealName, appNs)
```

Comment:
@thecooldrop , can you check for `case "tree=detailed"` as well?

Reply:
There was a same error there as well. Fixed it.